src/Sylius/Bundle/ApiBundle/Resources/translations/validators.en.yaml

```diff
@@ -23,6 +23,7 @@ sylius:
         not_sufficient: 'The product variant with %productVariantCode% code does not have sufficient stock.'
         product_variant_with_name_not_sufficient: 'The product variant with %productVariantName% name does not have sufficient stock.'
     shipment:
+        not_found: 'The shipment with %shipmentId% id does not exist.'
```

Comment:
Will this message be shown to the customer? If so, then maybe don't expose the shipment id and use the index instead, like in Twig templates: `shipment #1`, `shipment #2`, etc.

Reply:
It's designed with frontend developers in mind, and I think the validator messages are unlikely to be what they put directly on the page. In my opinion, it's feedback to the application designer on how to deal with using the API.